### ISSUE TITLE
ChromeCast: Play Queue

### ISSUE THREAD

**Issue by daneren2005** [daneren2005/Subsonic#516]
https://developers.google.com/cast/docs/autoplay

**Comment by keckelt** [comment]
I read that, with Google Play Music, if you cast a playlist to ChromeCast and leave (i'm not sure what was meant with leave here), it continues automatically. Is that the Autoplay feature?

**Comment by daneren2005** [comment]
Correct. Before you could only play a single song at a time but a while ago Chromecast added support for sending the whole queue so it could keep playing if the user lost connection.

**Comment by keckelt** [comment]
Cool, thanks for the explanation. Maybe I'll order the new Chromecast 
Audio when this has found it's way to DSub.

Am 09.10.2015 um 16:09 schrieb Scott Jackson:
